src/Options.ts

```diff
@@ -26,7 +26,7 @@ export namespace Options {
 
     /**
      * The maximum time the circuit remains open before doing a new attempt.
-     * Defaults to 60s
+     * Defaults to 30s
```

Comment:
The effective default was 30s per the code, so I used the actual value and updated documentation.